containers/hub-xmlrpc-api-image/Dockerfile

```
@@ -21,7 +21,8 @@ LABEL org.opencontainers.image.vendor="${VENDOR}"
 LABEL org.opencontainers.image.url="${URL}"
 LABEL org.opencontainers.image.version="5.0.0"
 LABEL org.openbuildservice.disturl="%DISTURL%"
-LABEL org.opensuse.reference="${REFERENCE_PREFIX}/server:5.0.0.%RELEASE%"
+LABEL org.opensuse.reference="${REFERENCE_PREFIX}/hub-xmlrpc-api:${PRODUCT_VERSION}.%RELEASE%"
```

I can't directly add a suggestion on the right line, but for tagging I also need one more  lable that is missing  in this Dockerfile
```suggestion
LABEL org.opensuse.reference="${REFERENCE_PREFIX}/hub-xmlrpc-api:${PRODUCT_VERSION}.%RELEASE%"
LABEL org.opencontainers.image.name=hub-xmlrpc-api-image
```